pkg/asset/asset.go

```diff
@@ -37,6 +37,9 @@ const (
 	AssetPathManifests                   = "manifests"
 	AssetPathKubelet                     = "manifests/kubelet.yaml"
 	AssetPathProxy                       = "manifests/kube-proxy.yaml"
+	AssetPathProxySA                     = "manifests/kube-proxy-sa.yaml"
+	AssetPathProxyRoleBinding            = "manifests/kube-proxy-role-binding.yaml"
+	AssetPathServiceAccountKubeConfig    = "manifests/kubeconfig-sa.yaml"
```

Comment:
Sorry, my nit doesn't help much. Its still confusing to read since this is actually a secret. eh, better idea?

Reply:
in-cluster-kubeconfig? which happens to be a secret or configmap?

Reply:
Does `manifest/kubeconfig-in-cluster.yaml` work?

Reply:
sgtm